modules/nf-core/ashlar/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "ashlar"
+description: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration
+keywords:
+  - image_processing, alignment, registration
+tools:
+  #
+  - "ashlar":
+      description: "Alignment by Simultaneous Harmonization of Layer/Adjacency Registration"
+      homepage: "https://labsyspharm.github.io/ashlar/"
+      documentation: "https://labsyspharm.github.io/ashlar/"
+      doi: "10.1101/2021.04.20.440625"
```

Comment:
Final non-preprint DOI is `10.1093/bioinformatics/btac544`

Reply:
updated that.  Thanks!